### DISCUSSION TITLE
ControlNet v1.1.232 can only use one unit?

### DISCUSSION THREAD

**Discussion by cocoshe** [Mikubill/sd-webui-controlnet#1782] [Q&A] [answered]
![image](https://github.com/Mikubill/sd-webui-controlnet/assets/69197635/624451b2-dcb6-4541-8e4f-1059b51d779e)

How to add units on it?

**Comment by ostap667inbox** [comment, marked as answer]
Settings > ControlNet

**Comment by PaxExterminatus** [comment]
Which option is responsible for this?

**Reply by ostap667inbox** [reply]
Multi ControlNet: Max models amount
